### PR TITLE
Add labels to improve accessibility

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
Adding labels to inputs/selects greatly increases accessibility by linking the text to the input element

Preference Page (before changes):
image

Preferences Page (after changes):
image

### CODE REVIEW

File: src/components/FeedPage.vue [hunk 1]
```diff
@@ -18,8 +18,8 @@
     </span>
 
     <span class="uk-align-right">
-        {{ $t("actions.sort_by") }}
-        <select class="uk-select uk-width-auto" v-model="selectedSort" @change="onChange()">
+        <label for='ddlSortBy'>{{ $t("actions.sort_by") }}</label>
```
Comment (FireMasterK) [inline src/components/FeedPage.vue:21]: You've used a mixture of single quotes and double-quotes. You should be using only double quotes. If you are using prettier, there is a config that you can import to do this when formatting :)

**Review by FireMasterK — CHANGES_REQUESTED** [review]
Just had a look, everything else seems fine from a glance.

File: src/components/FeedPage.vue (hunk 1 above)
Comment (ChunkyProgrammer) [reply src/components/FeedPage.vue]: Sorry about that, should be fixed now

**Comment by FireMasterK** [comment]
Thank you!

**Pull request merged**